Update README.md

rvdgraaf opened the pull request:
# Describe your changes

Removed semicolon from sc.exe command

## Type of change

Please check only ONE option.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How was this tested?

Documentation cannot be tested.

# Checklist before requesting a review
- [ ] I have reviewed and tested my changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I selected `develop` as the base branch and not `main`, or the pull request is a hotfix that needs to be done directly on `main`
- [X] I double checked all my changes and they contain no temporary test code, no code that is commented out and no changes that are not part of this branch
- [ ] I added new unit tests for my changes if applicable

# Related pull requests

N/A

# Link to Asana ticket

N/A

HansHappyHorizon reviewed (changes requested):
Please change the branch to `develop`, otherwise we have to merge the `main` branch into `develop` so a merge between these two branches has to happen anyway and the usual order is from `develop` to `main`.

And purely out of curiosity: Why was the semicolon removed in that script? Does it not work if you add a semicolon?

rvdgraaf commented:
Changed base from main to develop. I thought that the readme.md could be merged directly with main. 

With semicolon it doesn't work indeed. I experienced this problem several times, that's why I changed it in the documentation.
